### ISSUE TITLE
Unable to set the site's description to the numbers 1 or 0

### ISSUE THREAD

**Issue by Marky-Gee** [TryGhost/Ghost#11759]
### Issue Summary
Setting site description to the numbers 1 or 0 will get changed to true or false on save. If the value is true, it will display on website, if false it will not.

### To Reproduce

1. Open Ghost admin > General Settings
2. Expand Title & Description
3. Edit the description to either 1 or 0
4. Save

### Technical details:

* Ghost Version: 3.13.4 & 3.12
* Node Version: 10.15.3
* Browser/OS: Chrome 
* Database: Sqlite & MySql

**Comment by Marky-Gee** [comment]
Actually it seems none of the text inputs on General Settings can be set to 0 or 1

**Comment by cheepo2109** [comment]
Hi @ErisDS!
I think I've found the reason why this is happening.

Settings input serializer is checking for 0 and 1 and updates them.
```
// CASE: Ensure we won't forward strings, otherwise model events or model interactions can fail
      if (setting.value === '0' || setting.value === '1') {
             setting.value = !!+setting.value;
       }

```
I checked on the schema and looks like all the fields in the settings are strings and don't need to be booleans.
I'm not sure if it's a good idea to remove those checks, because probably there was a reason for them to be there in the first place 😅
